README.md

```diff
@@ -85,6 +85,29 @@ Here is a quick list of what's kept the same or what's similar between `content`
   - [Shell prompt guidelines](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/Shell)
 - **Formatting and layout:** the translated docs should, in general, be formatted and laid out in the same way as the en-US versions — however, if you find mistakes (e.g. incorrect Markdown syntax), please correct them in the translated file(s), and feel free to submit a PR at the mdn/content repo to fix those errors at the source as well!
 
+#### Unicode characters
+
+Code editors like VS Code have implemented features to highlight characters that are "often confused with another" and/or "invisible" (in VS Code, this is done with a yellow rectangle surrounding a character) — you can hover over each highlighted character to see its unicode hex and a potential replacement. The [Gremlins tracker for Visual Studio Code] extension ([find/install it here](https://marketplace.visualstudio.com/items?itemName=nhoizey.gremlins)) is an excellent to search for these characters as well — see after the table below for images of the VS Code and Gremlins tracker features in action. Here are some examples of "gremlins":
+
```

Comment:
Clarify that these chars aren't detected on the GitHub UI, and that they're not too concerning

```suggestion

> **Note:** These "invisible" chars, or "gremlins", aren't detected or highlighted on the GitHub UI, so you'll have to check for some by reviewing locally. Don't worry too much about gremlins, though it won't hurt to nitpick a little with typography.

```